### ISSUE TITLE
Fabric port instantly crashes server. 

### ISSUE THREAD

**Issue by ShikuTeshi** [jaredlll08/Clumps#86]
I tried using the recently released 1.18 fabric version of Clumps in a server environment only to have it instantly crash.

Unfortunately the logs don't get anything written down beyond initially loading the list of mods which I've tried with Clumps + Fabric API and Clumps alone to no success.

**Comment by jaredlll08** [comment]
I'm unable to reproduce, I need *some* logs, it shouldn't just crash with no logs.

What I've tried to reproduce:
Starting a dev server (no crash)
Starting a production server (no crash)

What fabric version are you using?
What java version are you using?

**Comment by ShikuTeshi** [comment]
I think the current server is running Fabric 12.2. It was created quite soon after 1.18 dropped.
Also running Java 17.0.1 JDK.
There's actually no logs beyond initially trying to load the .jar
This is what I get when I tried to grab a screenshot of the CMD before it closes within the half second it's opened for.
![image](https://user-images.githubusercontent.com/15152750/144646284-417c90a6-eced-49f5-81c5-3b559397d76d.png)

Given that you aren't having any issues I decided to give it a try with a fresh fabric server using 12.8 that's currently out and it had no issues.
I just tried updating the existing server from 12.2 to 12.8 and it appears to be working now.
